src/server/views/payment/index.njk

```diff
@@ -18,7 +18,13 @@
             <h2 class="heading-medium error-summary-heading" id="error-summary-payment-code">
               {{ t.payment_code_page.error.title }}
             </h2>
-             {{ t.payment_code_page.error.description }}
+            <ul class="error-summary-list">
+              <li>
+                <a href="#payment_code">
```

Comment:
What's the goal of this link? Is it some nunjucks syntax I'm not familiar with? Or do you really want to do a hash navigation?

Reply:
It links to the payment code input as per the examples here: https://design-system.service.gov.uk/components/error-summary/

Reply:
Hmm, not sure I fully understand that mechanism even after looking at those examples, but fine as long as it works!